sigs.yaml

```diff
@@ -2324,6 +2320,10 @@ sigs:
     - github: youngnick
       name: Nick Young
       company: Isovalent
+  - name: gateway-api-inference-extension
+    description: Gateway API Inference Extension
+    owners:
+    - https://raw.githubusercontent.com/kubernetes-sigs/llm-instance-gateway/main/OWNERS
```

Comment:
TODO: fixup this link now that the repo was renamed?

(this link works, but we should use the canonical name)

Reply:
https://github.com/kubernetes/community/pull/8218